### DISCUSSION TITLE
关于 GitHub 疯传的计算机病毒

### DISCUSSION THREAD

**Discussion by sulingjiang** [Meloong-Git/PCL#4616] [讨论]
各位尽量避免在回复中包含正文内容，只需要发个是计算机病毒就行
wuliaodexiaoluo 因为这问题惨遭牵连，目前已经被 GitHub 误删账号了

**Comment by wyc-26** [comment]
谢谢提醒，但是晚了，已经被 ban 了
大号：wuyuncheng-26

**Reply by wyc-26** [reply]
不是你们别欢呼啊（悲
![image](https://github.com/user-attachments/assets/2f2d0318-6e8d-4194-a6ce-883f97927b8b)

**Reply by wyc-26** [reply]
![image](https://github.com/user-attachments/assets/ca716a38-136b-4014-b95d-ee0e8b8a5664)
不是哥们

**Comment by kong-que-dnf123** [comment]
虽然但是不明不白的我啥都没明白
什么有病毒啊说文件有病毒下面怎么还有人直接引用的啊....

**Reply by Deep-Dark-Forest** [reply]
大致意思就是好几个账号，疯狂地在知名的仓库中的 issue 评论区里发诱导你下载病毒的评论，其中包括病毒下载链接
如果你引用并回复这个评论，即使是你把评论中的病毒下载链接改掉了发出去（除了下载链接以外的内容不变）号也会被 ban

**Reply by sulingjiang** [reply]
>虽然但是不明不白的我啥都没明白
什么有病毒啊说文件有病毒下面怎么还有人直接引用的啊....

小洛的说法是为了消歧义（虽然我也不知道有啥好消的）

**Reply by 3gf8jv4dv** [reply]
You can find some examples in #4597.

Sometimes, when a user submits an issue in a certain repo, such comments may appear immediately. Judging from the personal pages of some accounts, it looks more like they were infected by a virus and their accounts were stolen, and then the accounts were exploited to continue posting such comments.

In short, do not download or execute unknown programs easily, especially when their content has nothing to do with the issue or discussion topic (it is said that this also appears on Discord). If you see such comments when submitting an issue, it is also recommended to promptly remind users with relevant permissions in the repository to delete these comments.

**Reply by wuliaodexiaoluo** [reply]
>什么有病毒啊说文件有病毒下面怎么还有人直接引用的啊....

没有引用导致的歧义

![image](https://github.com/user-attachments/assets/091105f1-387c-4fe7-aa12-4ab15f890838)

**Comment by allMagicNB** [comment]
![1724493827663.png](https://github.com/user-attachments/assets/8a7c5ba3-3762-454e-9584-a36a13d0e747)
[背景音乐](https://music.163.com/#/song?id=1404596131)
今天我们怀着无比沉重的心情，聚集在这里，送别我们亲爱的 wuliaodexiaoluo 和 wuyuncheng-26。他们的离去让我们感到无比悲痛，但同时我们也深知，他们在世时留下的美好回忆和善行将永远伴随着我们。
wuliaodexiaoluo 和 wuyuncheng-26 一生为人正直，乐于助人，他们的爱与关怀洒遍了身边的每一个人。他们的智慧和善良，他们的坚韧与勇气，永远铭刻在我们的心中。
虽然他们已经离我们而去，但他们的一生将成为我们前行的灯塔，指引我们走向更加美好的未来。在此，我们也希望大家能够节哀顺变，将悲痛化为力量，继续走好人生的每一步。
愿 wuliaodexiaoluo 和 wuyuncheng-26 的 ghost 在另一个世界得到安息。我们会永远怀念他们，也会在心中保留他们最美好的回忆。

**Reply by sulingjiang** [reply]
>今天我们怀着无比沉重的心情，聚集在这里，送别我们亲爱的 wuliaodexiaoluo 和 wuyuncheng-26。他们的离去让我们感到无比悲痛，但同时我们也深知，他在世时留下的美好回忆和善行将永远伴随着我们。
wuliaodexiaoluo 和 wuyuncheng-26 一生为人正直，乐于助人，他们的爱与关怀洒遍了身边的每一个人。他们的智慧和善良，他们的坚韧与勇气，永远铭刻在我们的心中。
虽然他们已经离我们而去，但他的一生将成为我们前行的灯塔，指引我们走向更加美好的未来。在此，我们也希望大家能够节哀顺变，将悲痛化为力量，继续走好人生的每一步。
愿 wuliaodexiaoluo 和 wuyuncheng-26 的 ghost 在另一个世界得到安息。我们会永远怀念他们，也会在心中保留他们最美好的回忆。

小洛看到了，让我带一句话

>不是我复活赛还没打完你发这个。。。。我真的。。。。谢谢你

**Reply by wyc-26** [reply]
指正：并不是 ghost，ghost 是被彻底删除的账户，而我们的这种只是暂时的 ban，可以申诉的
![865c6f58b00b98f3658e21798fa50c68](https://github.com/user-attachments/assets/4378bc7a-d3f0-4bcd-986f-17296d2daf5d)

**Comment by wuliaodexiaoluo** [comment]
解封了，下次不管这种闲事了，没好处还惹得一身骚

**Reply by wyc-26** [reply]
~你是解封最速~
剩下二位就可怜了

**Comment by wyc-26** [comment]
<details><summary>大图</summary>
<p>

![image](https://github.com/user-attachments/assets/51183159-0b93-4a01-92d5-d3cea6309ff6)

![image](https://github.com/user-attachments/assets/a80f969d-5687-4549-8ec6-bc9a551555eb)

</p>
</details>
让我们恭喜柚子成为第三位受害者 🤣

**Reply by wuliaodexiaoluo** [reply]
本次事件造成了三人账号被 ban，共计两名贡献者，一名协作者
~解封最速~

**Comment by allMagicNB** [comment]
![1724493827663.png](https://github.com/user-attachments/assets/8a7c5ba3-3762-454e-9584-a36a13d0e747)
[背景音乐](https://music.163.com/#/song?id=1404596131)
今天我们怀着无比沉重的心情，聚集在这里，送别我们亲爱的 youzi-2333 和 wuyuncheng-26。他们的离去让我们感到无比悲痛，但同时我们也深知，他们在世时留下的美好回忆和善行将永远伴随着我们。
youzi-2333 和 wuyuncheng-26 一生为人正直，乐于助人，他们的爱与关怀洒遍了身边的每一个人。他们的智慧和善良，他们的坚韧与勇气，永远铭刻在我们的心中。
虽然他们已经离我们而去，但他们的一生将成为我们前行的灯塔，指引我们走向更加美好的未来。在此，我们也希望大家能够节哀顺变，将悲痛化为力量，继续走好人生的每一步。
愿 youzi-2333 和 wuyuncheng-26 的 ghost 在另一个世界得到安息。我们会永远怀念他们，也会在心中保留他们最美好的回忆。

**Reply by wuliaodexiaoluo** [reply]
6

**Reply by wyc-26** [reply]
不是哥们
你又想要被标 off-topic 吗（

**Comment by wyc-26** [comment]
<details><summary>超大图请注意 ⚠</summary>
<p>

![105406463fb38ade3396dccad65e9e60](https://github.com/user-attachments/assets/5c0e5f76-7c6d-4ae3-b284-b5c2cc974b7a)

</p>
</details>
1 群集体默哀

**Reply by youzi-3222** [reply]
嗯……不要缅怀我了！

还得等 GitHub 处理。估计要像你一样彻夜难眠了

**Reply by allMagicNB** [reply]
![410 Gone](https://github.com/user-attachments/assets/8a7c5ba3-3762-454e-9584-a36a13d0e747)
[背景音乐](https://music.163.com/#/song?id=1404596131)
今天我们怀着无比沉重的心情，聚集在这里，送别我们亲爱的 youzi-2333。他的离去让我们感到无比悲痛，但同时我们也深知，他在世时留下的美好回忆和善行将永远伴随着我们。
youzi-2333 一生为人正直，乐于助人，他的爱与关怀洒遍了身边的每一个人。他的智慧和善良，他的坚韧与勇气，永远铭刻在我们的心中。
虽然他已经离我们而去，但他的一生将成为我们前行的灯塔，指引我们走向更加美好的未来。在此，我们也希望所有的亲友能够节哀顺变，将悲痛化为力量，继续走好人生的每一步。
愿 youzi-2333 的 ghost 在另一个世界得到安息。我们会永远怀念他，也会在心中保留他最美好的回忆。

**Reply by wyc-26** [reply]
> 嗯……不要缅怀我了！
> 
> 还得等 GitHub 处理。估计要像你一样彻夜难眠了

可别提了，我今天凌晨 3:37 醒了，就睡了四个半小时
想着打开 QQ 看一下有没有人艾特我说解封了
结果根本没有（当时的时间对应的是美国时间下午四点左右）

**Comment by wyc-26** [comment]
<details><summary>图片</summary>
<p>

![image](https://github.com/user-attachments/assets/9016f7a9-daae-4d4e-b07e-d1e527fd1552)

![image](https://github.com/user-attachments/assets/cc77fcb0-816b-4a40-9873-12331a81b001)

![image](https://github.com/user-attachments/assets/69fe7881-04ec-40f4-8123-5e7c8a218a4e)

![image](https://github.com/user-attachments/assets/8187a913-0060-49d9-b315-3b8f01e58ebe)

</p>
</details>

对 GitHub 彻底失望了，连开发者都误 ban，而且这申诉基本都没用的，回复还慢的要死

**Comment by wyc-26** [comment]
附：目前 3gf8 的账号也由于未知原因被 GitHub ban 了
![IMG_6546](https://github.com/user-attachments/assets/29d71cf8-3152-40e8-b422-af1399abc96c)
当然这个可能不是因为病毒（

**Reply by wuliaodexiaoluo** [reply]
看来 ms 喜欢吃账号（确信

**Comment by wyc-26** [comment]
![image](https://github.com/user-attachments/assets/cf4bf06d-1eb3-42b8-93c4-7bdd5ee61fb4)

**Comment by Minemetero** [comment]
现在是2025年了。。。我才看到这里在讨论。我去年也中这了这个病毒。变成Ghost 账号了，不过GIthub帮我找回来了。效率也是很高的。

那个变Ghost的账号就是我现在的这个

**Reply by Minemetero** [reply]
我没记错的话是Win12 的那个repo中的

**Reply by LinQingYuu** [reply]
> 现在是2025年了。。。我才看到这里在讨论。我去年也中这了这个病毒。变成Ghost 账号了，不过GIthub帮我找回来了。效率也是很高的。
> 
> 那个变Ghost的账号就是我现在的这个

你是效率高了，但是现在还有两个人没恢复，两人工单还是去年交的（

**Comment by wyc-26** [comment]
新消息，不过是坏的……
这个处理态度我没话说了
以后就用这个号了，要开始疯狂交 PR 了（
![image](https://github.com/user-attachments/assets/f7a9a264-ff91-41f4-abd1-6a664af6fa76)

**Reply by youzi-3222** [reply]
我的也被 Archive 了。回复一毛一样。哈哈哈哈哈。

![image](https://github.com/user-attachments/assets/728f4330-99cb-4885-a84a-f60d7b883524)

**Reply by LinQingYuu** [reply]
可以多来点 Ticket，可能刚好就遇上了态度不那么差的 Staff，给你解封了也说不定

不过这个处理态度也是神了
